rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_07_22.md

```diff
@@ -0,0 +1,140 @@
+# Proposal to upgrade the ledger suite orchestrator canister
+
+Git hash: `de29a1a55b589428d173b31cdb8cec0923245657`
+
+New compressed Wasm hash: `81f426bcc52140fdcf045d02d00b04bfb4965445b8aed7090d174fcdebf8beea`
+
+Target canister: `vxkom-oyaaa-aaaar-qafda-cai`
+
+Previous ledger suite orchestrator proposal: https://dashboard.internetcomputer.org/proposal/131053
+
+---
+
+## Motivation
+
+This upgrades the ledger suite orchestrator and all managed ledger suites to the latest version.
+
+The motivation to upgrade the orchestrator are mainly the following points:
+1. Add logic to upgrade managed canisters.
```

Comment:
I think this sentence is a bit (too) ambiguous. Is logic added to make upgrades possible or is the existing upgrade logic extended?

Reply:
When we released it, we didn't have yet the logic to upgrade managed canisters. So how about
```suggestion
1. Add logic to make upgrade of managed canisters possible.
```